send-transaction-service/src/transaction_client.rs

```diff
@@ -249,6 +267,7 @@ impl TpuClientNextClient {
         leader_info: Option<T>,
         leader_forward_count: u64,
         identity: Option<&Keypair>,
+        bind_socket: UdpSocket,
```

Comment:
maybe call it `client_socket` or just `socket`? There is nothing particularly special about it.

Reply:
we use consistently "bind" as a term for the client side socket here and in other places. I would like to stick with that for consistency.